src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -1405,13 +1437,32 @@ namespace rascal {
     void compute_impl(std::shared_ptr<StructureManager> manager);
 
    protected:
+    //! cutoff radius r_c defining the size of the atom centered environement
     double interaction_cutoff{};
+    //! size of the transition region r_t spanning [r_c-r_t, r_c] in which the
+    //! contributions to the environment expansion go to zero smoothly
     double cutoff_smooth_width{};
+    //! defines the maximal mean error allowed to the interpolator when fitting
+    //! the reference
     double interpolator_accuracy{};
+    //! number of radial basis function to use in the expansion
     size_t max_radial{};
+    /**
```

Comment:
Not really important, but why use different doc comment styles ? This line uses `/** doc */` and below uses `//! Doc`. I am fine with either, but maybe we should be consistent and stick to one style.

Reply:
I use `//!` when its a one liner and `/**` for longer comments. Then when you already have a one liner and you add a bit of text I keep the `//!` when its still 2 lines. I am not sure there is a need to stick to one in particular is there ?

Reply:
I prefer when the style is consistent, it makes it easier to navigate the codebase. But this is a minor point w.r.t this PR